pyrenew/convolve.py

```diff
@@ -165,3 +165,45 @@ def _new_scanner(
         return latest, (new_val, m_net1)
 
     return _new_scanner
+
+
+def compute_delay_ascertained_incidence(
+    p_observed_given_incident: ArrayLike,
+    latent_incidence: ArrayLike,
+    delay_incidence_to_observation_delay_pmf: ArrayLike,
```

Comment:
```suggestion
    delay_incidence_to_observation_pmf: ArrayLike,
```
throughout

Reply:
sorry! I will go through it again

Reply:
fixed in https://github.com/CDCgov/PyRenew/pull/398/commits/4bc1736d0e1fd2485975cb9d18a44e021896fa8d